.github/workflows/build.yaml

```diff
@@ -33,52 +33,69 @@ jobs:
         strategy:
             matrix:
                 php:
-                    - '7.3'
                     - '7.4'
                     - '8.0'
+                    - '8.1'
+                dependencies: [highest]
+                symfony: ['*']
+                validate_docs: [false]
+                with_imagine: [false]
+                with_oneupbundle: [true]
                 include:
-                    - description: 'Lowest'
-                      composer_option: '--prefer-lowest'
+                    - extra: ', validate docs with lowest deps on PHP 7.4'
+                      php: '7.4' # PHP 8 not yet supported by rusty
+                      symfony: '*'
                       validate_docs: true
+                      dependencies: lowest
                       with_imagine: false
                       with_oneupbundle: false
-                    - description: '7.4'
-                      php: '7.4'
-                      with_imagine: true
-                      validate_docs: false
-                      with_oneupbundle: true
-                    - description: '8.0'
-                      php: '8.0'
-                      # needed because doctrine/mongodb-odm doesn't support php8 yet
-                      composer_option: '--ignore-platform-reqs'
-                      symfony: 5.2.*
-                      validate_docs: false
+                    - php: '8.0'
+                      symfony: '5.4.*'
+                      dependencies: highest
                       with_imagine: false
                       with_oneupbundle: true
-        name: PHP ${{ matrix.php }} tests (${{ matrix.description }})
+
+# Cannot be enabled yet because knplabs/knp-gaufrette-bundle doesn't support SF6 yet
```

Comment:
Indeed, knplabs/knp-gaufrette-bundle supports sf6 in 0.7.x-dev

Reply:
Using that now, thanks